$NetBSD$

NetBSD does not have a gethostent(3) function, so let's simulate it.
Darcs does not use this function, so it should be pretty safe to return
an empty list of hosts.

--- gethostent.c.orig	Tue Jan  4 22:39:27 2005
+++ gethostent.c
@@ -0,0 +1,5 @@
+#ifdef __NetBSD__
+void *gethostent(void) { return (void *) 0; }
+#else
+typedef int dummy;
+#endif
